XamarinCommunityToolkitSample/Resx/AppResources.es.resx

```diff
@@ -422,4 +422,7 @@ It converts the entered text to Upper Case and checks that it is Not Equal to th
   <data name="EnableNested" xml:space="preserve">
     <value>#Enable nested:</value>
   </data>
+  <data name="ImpliedOrderGridDescriptionLabel" xml:space="preserve">
+    <value>#View's row and column are determined by the order they are added to the grid's children. RowSpan and ColumnSpan are honored.</value>
+  </data>
```

Comment:
Seems AutoGridLayoutBehaviorDescription is missed there, correct?

Reply:
I updated it to be the full description from this PR